bundlesize.config.json

```diff
@@ -10,27 +10,27 @@
     },
     {
       "path": "./packages/instantsearch.js/dist/instantsearch.production.min.js",
-      "maxSize": "76.25 kB"
+      "maxSize": "76.5 kB"
```

Comment:
The Hits UI component has to let widgets provide it with flavor specific components props. It contributes to not quite reducing the impact on this change for now. It will surely reduce the bundle size when those differences are no more.

To illustrate this, here is the main diff for React InstantSearch's UMD bundle (not minified):

![CleanShot 2024-02-14 at 17 13 16@2x](https://github.com/algolia/instantsearch/assets/154633/a9679b1e-2e60-4ef8-88c4-f3ff63203f0a)
![CleanShot 2024-02-14 at 17 12 04@2x](https://github.com/algolia/instantsearch/assets/154633/bbe19a3a-b5aa-42a2-ad24-bd1add2442f8)